### PR TITLE
CSM: Fix documentation error for register_on_*_chat_messages

### PR DESCRIPTION
**Pull request by DS-Minetest**
There's no `name` and an `s` was missing.

### CODE REVIEW

**Comment by Fixer-007** [comment]
Can you name your PR properly and add a description?

**Comment by DS-Minetest** [comment]
You are right, sorry.

File: doc/client_lua_api.md [hunk 1]
```diff
@@ -648,10 +648,10 @@ Call these functions only at load time!
       semi-frequent intervals as well as on server shutdown.
 * `minetest.register_on_connect(func())`
     * Called at the end of client connection (when player is loaded onto map)
-* `minetest.register_on_receiving_chat_message(func(name, message))`
+* `minetest.register_on_receiving_chat_messages(func(message))`
```
Comment (SmallJoker) [inline doc/client_lua_api.md]: This function is called for each new message. It would be better to correct the spelling in `s_client.cpp`, as only one message will be sent in argument 1. An additional line in the builtin code will keep the backwards compat: `core.register_on_receiving_chat_messages = core.register_on_receiving_chat_message`

Comment (nerzhul) [inline doc/client_lua_api.md]: no need for backward compat, we defined CSM API is not stable

Comment (SmallJoker) [inline doc/client_lua_api.md]: Well then, @DS-Minetest please change `s_client.cpp` to call `registered_on_receiving_chat_message` instead of the version with `s`, for consistency with the other callback functions.

Comment (DS-Minetest) [inline doc/client_lua_api.md]: Done. I hope I didn't forget anything. >_<

File: clientmods/preview/init.lua [hunk 2]
```diff
@@ -150,3 +150,9 @@ core.register_on_punchnode(function(pos, node)
 	return false
 end)
 
+core.register_chatcommand("privs", {
```
Comment (nerzhul) [inline clientmods/preview/init.lua]: This function is not related with PR

Comment (red-001) [inline clientmods/preview/init.lua]: looks like a rebasing failure.

**Comment by nerzhul** [comment]
merged & approved by @sfan5 n IRC
